bin/hawtio-proxy.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# This script does all the proxy magic we need to access hawtio on a cluster
+NGINX_PORT=${NGINX_PORT:-8085}
+FORWARD_PORT=${FORWARD_PORT:-9095}
+pod=$1
+if [ $# -lt 1 ]; then
+  echo "Usage: $0 <pod>"
+  exit 1
+fi
+if [ -z "$(which jq)" ]; then
+  echo "jq not found (hint: sudo dnf install -y jq)"
+fi
+nginx_config=$(mktemp)
+hawtio_base_path=/actuator/hawtio
+pod_json=$(oc get pod/$pod -o json)
+hawtio_path_override=$(echo "$pod_json" | jq -r '.spec.containers[].env[] | select(.name=="HAWTIO_BASE_PATH") | .value')
+if [ -n "$hawtio_path_override" ]; then
+  hawtio_base_path=$hawtio_path_override
```

Comment:
We might want to double quote `hawtio_path_override` here and on line 16.  I'm not sure if there's the potential for any special shell characters in there or not.  I would err on the side of quoting it myself.

Reply:
Done